docker-compose.yml

```diff
@@ -194,7 +194,7 @@ services:
   kopano_server:
     image: ${docker_repo:-zokradonh}/kopano_core:${CORE_VERSION:-latest}
     hostname: kopano_server
-    container_name: kopano_server
+    container_name: ${COMPOSE_PROJECT_NAME}-kopano_server
     depends_on:
       - db
```

Comment:
would't this change also require changes to the depends_on values_

Reply:
to answer myself, no it wouldn't since this value references the name of the container in compose, and not the value in container_name